src/util.js

```diff
@@ -3,7 +3,7 @@ define([
     'underscore',
     'jsdiff',
     'vellum/markdown',
-    'vellum/xpath',
+    'vellum/escapedHashtags',
```

Comment:
curious why this was renamed? this module handles standard xpaths, xpaths with hashtags, and xpaths with escaped hashtags. All of those are derivatives of xpath. It's not a big deal and I'm not requesting a rename, just wondering why it was changed?